assets/frontend/invoices/controllers.js

```diff
@@ -1,16 +1,40 @@
-(function() {
-    'use strict';
+(function () {
+  'use strict';
 
-    angular
-        .module('invoices.portal')
-        .controller('InvoicesController', InvoicesController);
+  angular
+    .module('invoices.portal')
+    .controller('InvoicesAddController', InvoicesAddController)
+    .controller('InvoiceUpdateController', InvoiceUpdateController)
+  ;
 
+  /* INVOICE CONTROLLER: displays the data of invoice from the service
```

Comment:
this description doesn't really explain the idea of the controller. this controller does/should not only displays the data from the invoice service